src/skin/legacyskinparser.cpp

```diff
@@ -1084,6 +1089,59 @@ QWidget* LegacySkinParser::parseCoverArt(QDomElement node) {
     return pCoverArt;
 }
 
+void LegacySkinParser::parseSingletonDefinition(QDomElement node) {
+    QString objectName = m_pContext->selectString(node, "ObjectName");
+    if (objectName.isEmpty()) {
+        SKIN_WARNING(node, *m_pContext)
+                << "SingletonDefinition requires an ObjectName";
+    }
+
+    QDomNode childrenNode = m_pContext->selectNode(node, "Children");
+    if (childrenNode.isNull()) {
+        SKIN_WARNING(node, *m_pContext)
+                << "SingletonDefinition requires a Children tag with one child";
```

Comment:
Ah no. We support only one child. What is the reason to call the node "Children"?